servers/rendering/rendering_device.cpp

```diff
@@ -168,8 +159,7 @@ void RenderingDevice::_free_dependencies(RID p_id) {
 		for (const RID &F : E->value) {
 			HashMap<RID, HashSet<RID>>::Iterator G = dependency_map.find(F);
 			ERR_CONTINUE(!G);
-			ERR_CONTINUE(!G->value.has(p_id));
-			G->value.erase(p_id);
+			ERR_CONTINUE(!G->value.erase(p_id));
```

Comment:
This will give a less clear message IMO, I'd say to add a message

Reply:
The previous error was obvious "...has() is false", this one isn't obvious, you'd have to know that `erase` returns false if kit present, the message should probably be something like "attempted to erase non-existing dependency, bug?"